### DISCUSSION TITLE
[提问] 关于 Issue 引用方式

### DISCUSSION THREAD

**Discussion by LinQingYuu** [Meloong-Git/PCL#5239] [提问] [answered]
![image](https://github.com/user-attachments/assets/48dc73e6-a646-4396-90f4-5abcbf0041e9)

这是怎么做到的？

**Comment by hejiehao** [comment, marked as answer]
`- https://github.com/Hex-Dragon/PCL2/issues/3284`

**Reply by LinQingYuu** [reply]
- https://github.com/Hex-Dragon/PCL2/issues/3284

~还得加个减号才行（~

**Comment by hejiehao** [comment]
- https://github.com/Hex-Dragon/PCL2/issues/3284
